### ISSUE TITLE
Crash with Create Worldshaper

### ISSUE THREAD

**Issue by vivvyinks** [LambdAurora/LambdaBetterGrass#91]
Attempting to pick-block with a Create Worldshaper on a better grass block will instantly crash the game on Fabric. 
[crash-2023-08-02_12.41.42-client.txt](https://github.com/LambdAurora/LambdaBetterGrass/files/12244094/crash-2023-08-02_12.41.42-client.txt)

**Comment by NolanHewitt** [comment]
Can confirm this issue happens to me as well.

**Comment by Hunter-495** [comment]
Yep, the second the worldshaper selects a grass block, the game hard crashes.

**Comment by SlayerTheChikken** [comment]
https://github.com/TeamChikken/LambdaBetterGrass-CHKNFork-1.19.2/commit/ae0c1b32c8cbaa992ab8cf7eb3fb46debec05ebf

**Comment by SlayerTheChikken** [comment]
Let me know if anyone wants the binary uploaded, this issue is quite old

**Comment by LambdAurora** [comment]
This bug has been fixed at some point during the 1.20 release cycle.
